eclipse/sw360
# feat(ui-rest): Changes in Release information

> Changes in Release information  
>   - change title "Download URL" to "Source Code Download URL"
>   - add new data field "Binary Download URL"
>   - added null check in change log for merge release

> * Which issue is this pull request belonging to and how is it solving it? (*#964*)
> * Did you add or update any new dependencies that are required for your change? - No

### How To Test?
> Get/Create/Update Release from UI and Rest. Merge Releases and verify
>   - "Download URL" should be reflected as "Source Code Download URL"
>   - New data field "Binary Download URL" should be added
>   - Export/Import Component
>   - Download Component excel sheet from Component tab

> Have you implemented any additional tests? - No

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
